net/nfs-kernel-server/patches/200-fix-macos-build.patch

```diff
@@ -0,0 +1,40 @@
+fixes two issue: prctl in configure and stat64 for modern macos 
+versions (including macos arm64)
```

Comment:
Can you send this to upstream?

Reply:
I have found really good solution to avoid ./cofigure patching. New patch doesn't have configure hack, it disables prctl() check via HOST_CONFIGURE_VARS

Reply:
I will send stat64 fix to upstream

Reply:
stat64 patch has been to upstream: https://bugzilla.linux-nfs.org/show_bug.cgi?id=372